### ISSUE TITLE
Use case: Progressive Enhancement

### ISSUE THREAD

**Issue by prushforth** [Maps4HTML/HTML-Map-Element-UseCases-Requirements#95]

**Comment by AmeliaBR** [comment]
Related to #32.

I'm not sure whether this makes sense as a separate use case, or if it should be integrated into every use case, in the same way that "accessibility" isn't a use case, it's a requirement for other use cases being met.

**Comment by prushforth** [comment]
Fair.  I was just thinking about sophisticated cartographic / spatial techniques that might be desirable, but would be such that they would be best left to scripting.  And if you make that decision, you need to probably think about what API you would need to achieve the technique.    For example, rendering vector tiles is unlikely to be taken on directly by browser engines (in the short term at least), but can certainly be done by script on canvas.  So you would need to design an API to support that, which is not so much about JS independence (per #32) as it is about what should be left to script (today's maps are 100% left to script or, as we've recently discovered, sophisticated server-side programming that is not really in scope for standardization.

Relatedly, you need to think about fallback, not just for when script fails to load, but even before that, for the situation where an older (future) browser doesn't support maps and layers at all, but may support JavaScript, how could you get that browser to give you a decent web map (using JavaScript, probably) and not break on the new markup.

**Comment by AmeliaBR** [comment]
> Relatedly, you need to think about fallback, not just for when script fails to load, but even before that, for the situation where an older (future) browser doesn't support maps and layers at all, 

Ok, well that's something that we can describe as a specific authoring use case: make it easy to write code that incorporates fallbacks (regardless of why those fallbacks are necessary).  Like how `<video>` allowed you to include a flash object as child content, but any browser that recognizes `<video>` knows just to ignore the fallback content.
